libraries/x86/test/test_x86_cmd.c

```diff
@@ -4,7 +4,7 @@
 #include "unity.h"
 #include "x86_cmd.h"
 
-static const char s_cmd[30];
+static char s_cmd[30];
```

Comment:
🤔
How did this work?

Reply:
IDK it segfaulted at me on my machine hence why I fixed it. I'm guessing newer `glibc` fixed something?

Reply:
Yeah, it really shouldn't have worked at all. It's a const char array that was being modified through strncpy.